test/framework/filetools.py

```diff
@@ -95,6 +95,7 @@ def test_extract_cmd(self):
             ('test.txz', "unxz test.txz --stdout | tar x"),
             ('test.iso', "7z x test.iso"),
             ('test.tar.Z', "tar xzf test.tar.Z"),
+            ('test.foo.bar.sh', "cp -a test.foo.bar.sh ."),
```

Comment:
Is there any chance that we would run such a command? Because it would fail...
```
[EESSI pilot 2020.12] $ cp -a modulerc.lua .
cp: ‘modulerc.lua’ and ‘./modulerc.lua’ are the same file
```

Reply:
Yes, this is required for https://github.com/easybuilders/easybuild-easyconfigs/pull/11982.

The generated extract command actually uses a template that is filled in with the full path to the source file (in the `sourcepath`), and is executed in the build directory.

So the actual command would actually look like "`cp -a /sources/t/test/test.foo.bar.sh .`" after a "`cd %(builddir)s`"

Reply:
Yeah, that's what I thought, just wanted to be sure